### ISSUE TITLE
find in files progress winodow - is always on top

### ISSUE THREAD

**Issue by rafi-d** [notepad-plus-plus/notepad-plus-plus#5367]
### Steps to Reproduce the Issue

1. 
2. 
3.

**Comment by Faq** [comment]
your other mentioned thing is dublicate of #4732

**Comment by rafi-d** [comment]
@Faq : right, thanks for linking to that other thread. I might add my suggestion to have an option to revert to  status like for both modal mode and preview the count.
